### PR TITLE
Use NailGun 0.18.2

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 REQUIRES = [
     'ddt',
     'fauxfactory',
-    'nailgun==0.18.1',
+    'nailgun==0.18.2',
```
Comment (elyezer) [inline setup.py]: Shouldn't we use nailgun<0.19 here?

**Comment by elyezer** [comment]
ACK

**Comment by Ichimonji10** [comment]
ACK

Let's follow up on the versioning discussion today.

File: setup.py (hunk 1 above)
Comment (sthirugn) [inline setup.py]: @elyezer if you think it works, please go ahead and submit a PR.

**Pull request merged**